content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst

```diff
@@ -2,62 +2,60 @@
-journal in which the landed cost accounting entries will be recorded.
+First, you need to go in :menuselection:`Inventory --> Configuration --> Settings` and activate the
+landed costs feature. You can also determine the default journal in which the landed cost
+accounting entries will be recorded.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_01.png
-    :align: center
+.. image:: integrating_landed_costs/landed-costs-setting.png
+   :align: center
+   :alt: Activate the landed cost feature in Inventory Settings
 
 Add costs to products
 =====================
 
 Receive the vendor bill
 -----------------------
 
-Let’s imagine I receive a bill from custom duties for a shipment. I’ll
-tick the box *Landed Costs* on the vendor bill line.
+Let's imagine I receive a bill from custom duties for a shipment. I'll tick the box *Landed Costs*
+on the vendor bill line.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_02.png
-    :align: center
+.. image:: integrating_landed_costs/landed-costs-field-vendor-bill.png
+   :align: center
+   :alt: Enable Landed Costs field on vendor bill line.
 
 .. note::
-         The landed cost product must be of type service.
+   The landed cost product must be of type service.
 
-If this product is always a landed cost, you can also define it on the
-product and avoid having to tick the box on each vendor bill.
+If this product is always a landed cost, you can also define it on the product and avoid having to
+tick the box on each vendor bill.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_03.png
-    :align: center
+.. image:: integrating_landed_costs/define-landed-cost-on-product.png
+   :align: center
+   :alt: Option to define a product as always a landed cost
 
-At the top of my vendor bill, I’ll see a button *create landed costs*.
-I click on this button and a landed cost is automatically created. I can
-now decide on which picking those additional costs should apply.
+At the top of my vendor bill, I'll see a button *create landed costs*. I click on this button and a
+landed cost is automatically created. I can now decide on which picking those additional costs
+should apply.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_04.png
-    :align: center
+.. image:: integrating_landed_costs/warehouse-transfer-landed-costs.png
+   :align: center
+   :alt: Using a warehouse transfer to cover a landed cost in the accounting journal.
 
-I can now click on *Compute* and go in the tab *Valuation
-Adjustments* to see the impact on my products costs. The last step is
-to validate the landed cost.
+I can now click on *Compute* and go in the tab *Valuation Adjustments* to see the impact on my
+products costs. The last step is to validate the landed cost.
 
-I can access the journal entry that has been created by the landed cost
-by clicking on the journal entry.
+I can access the journal entry that has been created by the landed cost by clicking on the journal
```

Comment:
@thomasdeleval this sentence is unclear. Is there a `Journal Entry` smartbutton that the user can click on? Or does the user have to navigate to the accounting journal to view the landed cost journal entry. 

I could not recreate the steps in Runbot to choose the picking, compute, and validate the landed cost, I got an error message `You cannot apply landed costs on the chosen Transfers(s). Landed costs can only be applied for products with FIFO or average costing method.` (Reference: https://mivu.nimbusweb.me/s/share/7482454/tkkn7m1pnvzkrvrul2b4 )

Reply:
@mivu-odoo 
https://drive.google.com/file/d/1g9MXA_qCWpikTU-i_QFkAz32PlK_wUcf/view?usp=drivesdk
The message is correct, the products on which to apply the landed cost must have a product category set in FIFO or AVCO.

Reply:
I suggested this for better clarity: https://github.com/odoo/documentation/pull/2556#discussion_r988411459